indica/equilibrium.py

```diff
@@ -34,24 +35,22 @@ class Equilibrium:
         A collection of equilibrium data rea in using
         :py:meth:`~indica.readers.DataReader.get_equilibrium`. TODO: List full set
         of required quantities.
-    R_shift : float
+    R_shift
         How much to shift the equilibrium inwards (or the remapped diagnostic outwards)
-        on the major radius.
-        TODO: this and z_shift should be time-dependent...
-    z_shift : float
+        on the major radius. Either a float for all time slices or a DataArray
+        with coord time
+    z_shift
         How much to shift the equilibrium downwards (or the remapped diagnostic upwards)
-        in the vertical coordinate.
-    offset_picker: OffsetPicker
-        A callback which determines by how much to offset the equilibrium data
-        along the major radius. Allows the user to select this interactively.
+        in the vertical coordinate. Either a float for all time slices or a DataArray
+        with coord time
 
     """
 
     def __init__(
         self,
         equilibrium_data: Dict[str, DataArray],
-        R_shift: float = 0.0,
-        z_shift: float = 0.0,
+        R_shift: Union[DataArray, float] = 0.0,
```

Comment:
I'd make a new datatype in indica.numpy_typing (call it something like FloatDataarray?) so you can import it and there's no need to write Union[..] in the other calls either.